frontend/src/pages/LearningCourse/Material/MaterialContainer.tsx

```diff
@@ -1,20 +1,19 @@
 import { FC } from 'react';
 
-import { MATERIAL } from 'constants/materials';
-import { optimizeLink } from 'utils/helpers/videoPlayer/videoLink';
 import { getPreviewId } from 'utils/helpers/videoPlayer/getPreviewId';
-import { defineMaterialType } from 'utils/helpers/defineMaterialType';
+import extractMaterial from 'utils/helpers/materials/extractMaterial';
+import { IContentElement } from 'types/course';
 
 import Material from './Material';
 
 interface IProps {
-  courseMaterial: string;
+  courseMaterial: IContentElement;
 }
```

Comment:
maybe it also should be moved to local types and with name like IMaterialContainer?

Reply:
The same. I don't think we will use this anywhere outside, even in Material component. But whatever, if you guys think that it would be better, I will agree with ya. 😺

Reply:
Here Irina suggest to create Local folder, not to describe interface in common types

Reply:
need to discuss again

Reply:
@k-krasilnikova got it

Reply:
done